addons/point_of_sale/static/src/js/models.js

```diff
@@ -173,18 +166,129 @@ class PosGlobalState extends PosModel {
     }
 
     async load_server_data(){
-        const loadedModels = await pos_env.services.rpc({
+        const loadedData = await pos_env.services.rpc({
             model: 'pos.session',
             method: 'load_pos_data',
             args: [[odoo.pos_session_id]],
-        })
-        const tmp = {};
-        for (const model of this.models) {
-            if (model.condition ? !model.condition(this) : false) continue;
-            await model.loaded(this, loadedModels[model.model] || [], tmp);
-        }
+        });
+        await this._processData(loadedData);
         return this.after_load_server_data();
     }
+   async _processData(loadedData) {
+        this.version = loadedData['version'];
+        this.company = loadedData['res.company'];
+        this.dp = loadedData['decimal.precision'];
+        this.units = loadedData['uom.uom'];
+        this.units_by_id = loadedData['units_by_id'];
+        this.states = loadedData['res.country.state'];
+        this.countries = loadedData['res.country'];
+        this.langs = loadedData['res.lang'];
+        this.taxes = loadedData['account.tax'];
+        this.taxes_by_id = loadedData['taxes_by_id'];
+        this.pos_session = loadedData['pos.session'];
+        this._loadPosSession();
+        this.config = loadedData['pos.config'];
+        this._loadPoSConfig();
+        this.bills = loadedData['pos.bill'];
+        this.partners = loadedData['res.partner'];
+        this._loadResPartner();
+        this.picking_type = loadedData['stock.picking.type'];
+        this.user = loadedData['res.users'];
+        this.pricelists = loadedData['product.pricelist'];
+        this.default_pricelist = loadedData['default_pricelist'];
+        this.bank_statement = loadedData['account.bank.statement'];
```

Comment:
we only need the first item.
```suggestion
        this.bank_statement = loadedData['account.bank.statement'][0];
```

Reply:
I gotchu
```diff
    def _get_pos_ui_account_bank_statement(self, params):
-        return self.env['account.bank.statement'].search_read(**params['search_params'])
+        return self.env['account.bank.statement'].search_read(**params['search_params'])[0]
```